FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentGaugeBar.cs

```diff
@@ -6,10 +6,93 @@ namespace FFXIVClientStructs.FFXIV.Component.GUI;
+    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    public struct GaugeValue {
+        public const int Size = 0x10;
+        [FieldOffset(0x0)] public int ValueInt; // true current value of the gauge
+        [FieldOffset(0x4)] public float ValueFloatIncreasing; // increases gradually to match others (but decreases instantly)
+        [FieldOffset(0x8)] public float ValueFloat;           // updates instantly to match ValueInt
+        [FieldOffset(0xC)] public float ValueFloatDecreasing; // decreases gradually to match others (but increases instantly)
+    }
+
+    /// <summary>
+    /// A set of three NineGrid nodes for the bar's fill level; one main fill node, and two nodes layered beneath in alternate colours.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    public struct GaugeFill {
+        public const int Size = 0x18;
+        [FieldOffset(0x00)] public AtkNineGridNode* MainFillNode;
+        [FieldOffset(0x08)] public AtkNineGridNode* IncreaseFillNode; // alt-colored fill shown during increases
+        [FieldOffset(0x10)] public AtkNineGridNode* DecreaseFillNode;                     // same, for decreases
+    }
+
+    // There are fields and functions to track two values per gauge, but the vast majority of gauges only use the first value
+    // The main (only?) use-case for the secondary gauge value is to display shields on Party List HP bars
+
+    [FixedSizeArray<GaugeValueTransitionData>(2)]
+    [FieldOffset(0xC0)] public fixed byte TransitionData[2 * GaugeValueTransitionData.Size];
+
+    [FixedSizeArray<GaugeValue>(2)]
+    [FieldOffset(0x100)] public fixed byte Values[2 * GaugeValue.Size];
+
+    [FieldOffset(0x120)] public AtkImageNode* BackdropImageNode;
+    [FieldOffset(0x128)] public GaugeFill PrimaryFill;
+
+    // optional nodes not seen on all bars
+    [FieldOffset(0x140)] public GaugeFill SecondaryFill;                // portion of shield that fits within the HP bar
+    [FieldOffset(0x158)] public GaugeFill SecondaryOverflow;            // portion of shield that overflows onto a separate bar
+    [FieldOffset(0x170)] public AtkImageNode* SecondaryOverflowMaxIcon; // the little "etc" icon that appears on a shield too big to display
+    [FieldOffset(0x178)] public AtkNineGridNode* RestedExpNode;         // specific to Exp bar, naturally
+    [FieldOffset(0x180)] public AtkNineGridNode* BorderNineGridNode;    // appears on crafting window bars
+    [FieldOffset(0x188)] public AtkTextNode* ParameterTextNode;         // mainly seems to be used by the Parameter Widget; other gauges that have a text node won't necessarily use this field to point to it
 
     [FieldOffset(0x190)] public int MinValue;
     [FieldOffset(0x194)] public int MaxValue;
+
+    [FieldOffset(0x1A0)] public short MarginX;
+    [FieldOffset(0x1A2)] public short MaxFillPositionX;
+
+    [MemberFunction("E8 ?? ?? ?? ?? 48 8B F8 48 85 C0 0F 84 ?? ?? ?? ?? 49 8B 0F")]
+    public partial void Ctor();
```

Comment:
Points to the wrong `ctor` currently is the sig of `140535D30` when it should be `14056F5E0`